server/todo-api/controllers/controller.ts

```diff
@@ -121,5 +121,30 @@ export class Controller {
       next(bodyErrors);
     }
   }
+
+  async getAll(req: Request, res: Response, next: NextFunction): Promise<void> {
+    const { createdBy } = req.body;
+
+    if (!isObjectId(createdBy)) {
+      const error: HTTPError = {
+        status: 400,
+        name: 'Bad request',
+        message: `createdBy must by a ObjectId value but recived a ${typeof createdBy}`,
+      };
+      next(error);
```

Comment:
Fail fast indica que se retorne inmediatamente cuando sucede un error con eso no hay necesidad de hacer el else y el codigo sigue un flujo mas limpio
```suggestion
      return next(error);
```